x-pack/test/security_solution_cypress/es_archives/exceptions_2/data.json

```diff
@@ -23,4 +23,31 @@
       ]
     }
   }
-}
\ No newline at end of file
+}
+
+{
+  "type": "doc",
```

Comment:
Do we strictly need 2 documents? If not, I would suggest extending the current document :)

Reply:
I added 2 because I wanted to update the tests to more explicitly show that the matching alert is not created and the non matching one is. Let me know if you think that's not necessary and I can update.